public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
- [ ] (Required) Good work 👏 implementing the change requested by the other reviewer. I would like to add that you remove this line of code from this file since you have deleted the `favicon` icon. 😄